central/policy/datastore/datastore_impl_postgres_test.go

```diff
@@ -211,3 +211,21 @@ func (s *PolicyPostgresDataStoreTestSuite) TestSearchPolicyCategoryFeatureDisabl
 	s.Len(results, 1)
 	s.Equal(policy.GetId(), results[0].ID)
 }
+
+func (s *PolicyPostgresDataStoreTestSuite) TestSearchRawPolicies() {
+	policy := fixtures.GetPolicy()
+
+	ctx := sac.WithGlobalAccessScopeChecker(context.Background(), sac.AllowFixedScopes(
+		sac.AccessModeScopeKeys(storage.Access_READ_ACCESS, storage.Access_READ_WRITE_ACCESS),
+		sac.ResourceScopeKeys(resources.Policy, resources.Cluster),
```

Comment:
why do you need cluster access to get policy?

Reply:
🤷‍♀️ that's what tests before had so I used the same